store.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Store</title>
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="store.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+        <script>
+            $(document).ready(function(){
+
+            });
+        </script>
+    </head>
+    <!--==================== HEADER ====================-->
+    <header class="header" id="header-store">
+        <div>
+            <img
+                src="img/karma-logo.svg"
+                alt="Karma official logo"
+                class="header_logo"
+            />
+        </div>
+    
+        <nav class="container">
+            <a class="nav_icon-text">Meet Karma</a>
+                
+            <a class="nav_icon">How it Works</a>
+    
+            <a class="nav_icon" href="index.html" target="_blank">Store</a>
+    
+            <a class="nav_icon">Blog</a>
+    
+            <a class="nav_icon">Help</a>
+    
+            <a class="nav_icon">Login</a>
+        </nav>
+    </header>
+
+    <!--==================== BODY ====================-->
+    <body class="store-body">
+
+    <div class="parent">
+
+        <div class="wrapper">
+            <div class="content">
```

Comment:
What happened to using semantic HTML? `main`, `section` etc. and very important here: the `<form>` element to wrap your form on line 52